doc/deployment_examples/sinfonier/feeding_sinfonier.md

```diff
@@ -0,0 +1,408 @@
+#<a name="top"></a>Sinfonier deployment of Cosmos
+Content:
+
+* [Introduction](#section1)
+* [Architecture](#section2)
+* [Orion contextBroker](#section3)
+* [Cygnus](#section4)
+    * [Configuration](#section4.1)
+    * [Running](#section4.2)
+* [Kafka](#section5)
+    * [Zookeeper](#section5.1)
+	   * [Configuration](#section5.1.1)
+	   * [Running](#section5.1.2)
+    * [Brokers](#section5.2)
+      * [Configuration](#section5.2.1)
+      * [Running](#section5.2.2)
+* [Sinfonier](#section6)
+* [General procedure step-by-step](#section7)
+* [Reporting issues and contact information](#section8)
+
+##<a name="section1"></a>Introduction
+This document describes how `Sinfonier` can consume stored context information from [Orion](http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker).
+
+The purpose is consuming information with `Sinfonier`. `Orion` has context information that could be useful, but we need a way to connect both elements. The use of `Cygnus` and `Kafka`, for traslate and storage the information in a data structure, show the connection between `Orion` and `Sinfonier`.
+
+[Top](#top)
+##<a name="section2"></a>Architecture
+The architecture for feeding `Sinfonier` needs some elements as shown in next image:
+
+![architecture][Architecture]
+
+The objective of this architecture is get a processed information from `Orion` to `Sinfonier`, using `Cygnus` for translate and `Kafka` for store the information that will be consumed.
+
+In next paragraphs all the needed configuration will be described in order to know how it works.
+
+[Top](#top)
+
+##<a name="section3"></a>Orion contextBroker
+First of all, [Orion](https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/admin/install.md) must be installed in the system. In addition, `Orion` needs `MongoDB` for storage, so must be installed too.
+`Orion` have context information that it's going to be translated through `Cygnus` to `Kafka`. Context information is stored in `Entities` with `Attributes`.
```

Comment:
Orion handles context information which is stored as entities holding attributes.

Tip: try to avoid explaining each time the data is going to be moved from Orion to Sinfonier ;)